toolchain/check/convert.cpp

```diff
@@ -837,6 +837,64 @@ auto ConvertToBoolValue(Context& context, Parse::Node parse_node,
       context.GetBuiltinType(SemIR::BuiltinKind::BoolType));
 }
 
+namespace {
+CARBON_DIAGNOSTIC(InCallToFunction, Note, "Calling function declared here.");
+}
```

Comment:
```suggestion
CARBON_DIAGNOSTIC(InCallToFunction, Note, "Calling function declared here.");
```

These are already static, shouldn't need a namespace.